src/handlers/serenity.rs

```diff
@@ -166,6 +166,32 @@ impl SerenityHandler {
                                         .required(true)
                                 })
                         })
+                        .create_option(|option| {
+                            option
+                                .name("reverse")
+                                .description("Add all tracks in reverse if the URL refers to a video and a playlist")
```

Comment:
Not quite, should work for both video/playlist links and simple playlist links.
For simplicity's sake, reword this to **"Add a playlist to the queue in reverse order"**.